data-collection/src/main/java/refactoringml/db/No.java

```diff
@@ -1,31 +1,43 @@
 package refactoringml.db;
 
+import refactoringml.util.JGitUtils;
 import refactoringml.util.RefactoringUtils;
 
 import javax.persistence.*;
 import java.util.Calendar;
 
+//TODO: create a Baseclass for both Yes and No, as they share a lot of logic
 @Entity
 @Table(name = "no", indexes = {@Index(columnList = "project_id"), @Index(columnList = "type"), @Index(columnList = "isTest")})
 public class No {
+	//TODO: candidate for a config file
+	private final int stringMaxLength = 255;
 
 	@Id
 	@GeneratedValue(strategy = GenerationType.IDENTITY)
 	private long id;
 
 	@ManyToOne
+	//project id: referencing the project information, e.g. name or gitUrl
 	private Project project;
 
+	//id of this commit
 	private String commit;
+	//original commit message
+	private String commitMessage;
+	//url to the commit on its remote repository, e.g. https://github.com/mauricioaniche/predicting-refactoring-ml/commit/36016e4023cb74cd1076dbd33e0d7a73a6a61993
+	private String commitUrl;
```

Comment:
This `commitUrl` on the other hand, I'd leave it here (instead of moving to the same class as the commit message). What do you think?